spec/controllers/applications/process_controller_spec.rb

```diff
@@ -144,6 +144,36 @@
     end
   end
 
+  describe 'GET #summary_save' do
+    let(:current_time) { Time.zone.now }
+    let(:user) { create :user }
+    let(:application) { create :application_full_remission }
+
+    before do
+      allow(Application).to receive(:find).with(application.id).and_return(application)
```

Comment:
You don't have to do this if you're creating the application above (line 150). This is needed only if we use `build_stubbed` for the main model, because then it's not stored in the db. If you could though change it to `build_stubbed`, that's better I guess.